src/constitutive_relations/column_integrators/PerchedWaterTableColumnIntegrator.hh

```diff
@@ -0,0 +1,113 @@
+/*
+  Copyright 2010-202x held jointly by participating institutions.
+  ATS is released under the three-clause BSD License.
+  The terms of use and "as is" disclaimer for this license are
+  provided in the top-level COPYRIGHT file.
+
+  Authors: Ethan Coon (coonet@ornl.gov)
+*/
+
+/*
+
+This is an evaluator for integrating arbitrary functionals across columns to
```

Comment:
Please update the documentation to describe what is being done here.  See, for example,  https://github.com/amanzi/ats/blob/master/src/constitutive_relations/column_integrators/ColumnSumEvaluator.hh#L13 which shows a typical documentation + input spec.  Note that you need to start this comment with `/*!` -- the ! tells sphinx to use this comment as the docstring.

Then, add an entry to https://github.com/amanzi/ats/blob/master/docs/documentation/source/input_spec/state/generic.rst.in for each new file in this directory, so something like:

```
Perched Water Table
^^^^^^^^^^^^^    <---- as many ^ as there are letters in the line above, emacs C-= does this for me
{ PerchedWaterTableColumnIntegrator }  <--- filename
```


This tells the documentation to read this file, looking for the "magic comment" that starts with /*!, and dump that comment as restructured text into the documentation.

{

Reply:
EDIT: ok, I see you have actually documented this to some degree in perched_water_table_evaluator.hh.

So the above still applies, just { perched_water_table_evaluator } instead...